Zilon.Core/Zilon.Core/PersonModules/CombatStatsModule.cs

```diff
@@ -25,5 +34,267 @@ public CombatStatsModule()
 
         /// <inheritdoc/>
         public bool IsActive { get; set; }
+
+        private void CalcCombatStats()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'CalcCombatStats'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5480188)